### ISSUE TITLE
[Bug]: Failed to sanitize DB2 connection string

### ISSUE THREAD

**Issue by weslleyshavo** [instana/nodejs#614]
### Problem Description

The sanitizeConnectionStr function regex doesn't match a connection string where PWD is the last parameter included and does not contain the semicolon. Package: tracingUtil.js
This causes string sanitize not to work.

### Short, Self Contained Example

Connection string: "DATABASE=MY_DATABASE;HOSTNAME=localhost;PORT=50000;PROTOCOL=TCPIP;UID=my_user;PWD=123456"

In this case the regex contained in the code does not match, as there is no semicolon.
**PWD=.*?(?=;)**
![image](https://user-images.githubusercontent.com/56171700/190714475-21d830b3-7396-486b-9684-594676c9248a.png)

I suggest using the regex like this:
**PWD=.*?(?=;|$)**
![image](https://user-images.githubusercontent.com/56171700/190714660-2b8aec69-5a5a-4e3e-a86f-abfccc978eb1.png)

### Node.js Version

Node.js v16.14.0

### package.json

```shell
{
  "name": "instana",
  "version": "1.0.0",
  "description": "",
  "main": "app.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "weslleyshavo",
  "license": "ISC",
  "dependencies": {
    "@instana/collector": "^2.7.1",
    "ibm_db": "^2.8.2"
  }
}
```

### package-lock.json

```shell
body is too long...
```

**Comment by basti1302** [comment]
Hey Weslley,

thanks for the detailed report. We'll take a look at this.

**Comment by basti1302** [comment]
A fix will be released with the next release (probably tomorrow).

**Comment by basti1302** [comment]
The fix has landed in version 2.8.0.
